MotionPhotoReader/app/src/main/java/com/google/android/libraries/motionphotoreader/MotionPhotoReader.java

```diff
@@ -147,6 +229,7 @@ private void startRenderThread(MotionPhotoInfo motionPhotoInfo) throws IOExcepti
         lowResExtractor.setDataSource(fd, file.length() - videoOffset, videoOffset);
 
         // Find the video track and create an appropriate media decoder
+        // TODO: Initialize high resolution media decoder (for stabilization)
```

Comment:
we won't need to stabilize the high-res track, so won't need a high-res mediadecoder.